src/Components/HugeProject/HugeProject.css

```diff
@@ -59,10 +58,46 @@
     color: white;
     font-size: small;
     font-weight: bolder;
-    height: 40px;
+    height: 2.5rem;
     position: absolute;
-    top: 150px;
+    top: 9.375rem;
     text-transform: uppercase;
     visibility: hidden;
-    width: 140px;
+    width: 8.75rem;
+}
+
+@media only screen and (max-width: 600px) {
+    .hugeProjectContainer {
+        height: 26.375rem;
+        width: 26.375rem;
+    }
+    .hugeProjectImg {
+        height: 185px;
+        width: 26.375rem;
+    }
+    .hugeProjectTitle {
+        font-size: 1.17rem;
+    }
+    .hugeProjectDescription {
+        font-size: 1rem;
```

Comment:
unless the original font size is already larger than the default changing it to 1rem won't do anything

Reply:
I see above that it is, nvm